charts/cadvisor/Chart.yaml

```diff
@@ -1,16 +1,16 @@
-apiVersion: v1
-description: A chart for a Cadvisor deployment
-name: cadvisor
-version: 1.1.1
-appVersion: 0.36.0
-home: https://github.com/google/cadvisor
-sources:
-  - https://github.com/google/cadvisor
-  - https://github.com/ckotzbauer/helm-charts
-keywords:
-  - prometheus
-  - cadvisor
-  - monitoring
-maintainers:
-  - name: ckotzbauer
-    email: christian.kotzbauer@gmail.com
+apiVersion: v1
+description: A chart for a Cadvisor deployment
+name: cadvisor
+version: 1.1.2
```

Comment:
I only changed version number. github workflow checks was complaining about the newline syntax, so I updated from CRLF to LF and saved this file again, it passed the check.